gcc/config/riscv/corev.md

```diff
@@ -2839,3 +2839,54 @@
   "cv.b%C1imm\t%2,%3,%0"
   [(set_attr "type" "branch")
    (set_attr "mode" "none")])
+
+;;CORE-V Post Increment Load/ Store
+;; Post Increment Register-Immediate and Register-Register Load/Store
+(define_insn "cv_load<mode>_postinc"
+   [(set (match_operand:ANYI 0 "register_operand" "=r")
+         (match_operand:ANYI 1 "mem_post_inc" "m"))]
+  "TARGET_XCVMEM && riscv_legitimate_post_inc_p (GET_MODE(XEXP(operands[1],0)) , XEXP(operands[1],0), (lra_in_progress || reload_completed))"
```

Comment:
`riscv_legitimate_post_inc_p (GET_MODE(XEXP(operands[1],0)) , XEXP(operands[1],0), (lra_in_progress || reload_completed))` could be done in`mem_post_inc` instead.